pkg/console/validator.go

```diff
@@ -274,6 +276,27 @@ func checkToken(token string) error {
 	return nil
 }
 
+func checkSystemSettings(systemSettings map[string]string) error {
+	if systemSettings == nil {
+		return nil
+	}
+
+	allowList := config.GetSystemSettingsAllowList()
+	for systemSetting, _ := range systemSettings {
```

Comment:
NIT: You don't necessarily need the `_` here if key is all you need:
```suggestion
	for systemSetting := range systemSettings {
```